var/spack/repos/builtin/packages/py-types-python-dateutil/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyTypesPythonDateutil(PythonPackage):
+    """Typing stubs for python-dateutil."""
+
+    homepage = "https://github.com/python/typeshed"
+    pypi = "types-python-dateutil/types-python-dateutil-2.8.19.tar.gz"
+
+    version("2.8.19", sha256="bfd3eb39c7253aea4ba23b10f69b017d30b013662bb4be4ab48b20bbd763f309")
+
+    depends_on("python@3.7:", type=("build", "run"))
```

Comment:
Where does 3.7+ come from?

Reply:
In the description [here](https://github.com/python/typeshed) (in the About section) they write that they only support Python versions 3.7 and up